tests/analysis.spec.ts

```diff
@@ -163,8 +163,8 @@ describe('Functional test of analysis', () => {
       expect(bundle.analysisResults.type === 'legacy').toBeTruthy();
       if (bundle.analysisResults.type !== 'legacy') return;
 
```

Comment:
[stepsize] Can we separate api contract testing from local unit tests?

Reply:
Next step: bring stepsize back to our lives :D

Reply:
Yes! 😬  One step at a time.